package.json

```diff
@@ -145,7 +145,16 @@
           "when": "editorLangId == scala"
         }
       ]
-    }
+    },
+    "debuggers": [
+      {
+        "type": "scala",
+        "label": "scala debugger",
```

Comment:
Does this label appear in the UI? If so, should it maybe be named "Scala Debugger" or "Metals Debugger"?

Reply:
done